layouts/html.hbs

```diff
@@ -7,6 +7,7 @@
         window.isOverlay = true;
       }
       {{> script/translations}}
+
```

Comment:
At first, I wanted to have it look like 
```{{ jsPartial 'script/translations' }}``` and have the jsPartial helper render the partial.
This isn't possible without reaching into the global handlebars instance, which I didn't want to do since the fact that we use the global hbs instance is kind of a jambo implementation detail, and we actually talked about having jambo not use the global instances and instead use Handlebars.create() and pass around an isolated instance.

Then, I tried out `{{#addNewLineToEnd}}{{> script/translations }}{{/addNewLineToEnd}}`, which I thought was a little ugly, but also don't mind this kind of approach if people prefer it. It's a lot more explicit. I intentionally what it would look like as the first commit to this pr.